docs/howto/run_material_mapping.rst

```diff
@@ -126,48 +142,32 @@ The same script can be used to dump a steering file that can help you selecting
 
 Material Mapping
 ----------------
 
-With the surfaces map and the material track we can finally do the material mapping using the ``MaterialMapping`` application:
+With the surfaces map and the material track we can finally do the material mapping using the ``material_mapping.py`` script:
 
 .. code-block:: console
 
-   $ <build>/bin/ActsExampleMaterialMappingDD4hep -j1 \
-       --input-root true \
-       --input-files geant4_material_tracks.root \
-       --mat-input-type file \
-       --mat-input-file geometry-map.json \
-       --output-root \
-       --output-json \
-       --output-cbor \
-       --mat-output-file material-maps \
-       --mat-mapping-surfaces true \
-       --mat-mapping-volumes true \
-       --mat-mapping-volume-stepsize 1 \
-       --mat-mapping-read-surfaces false \
-       --dd4hep-input <source>/thirdparty/OpenDataDetector/xml/OpenDataDetector.xml
-
-Note that technically when using DD4hep (in particular for the ODD) using the option ``--mat-input-type`` is not strictly necessary as the DD4hep geometry can hold the information of which surface to map onto with which binning. We will ignore this option, since the goal of this guide is to explain how to make a material map regardless of the detector.
+   $ python3 <source>/Examples/Scripts/Python/material_mapping.py 
+
+Note that technically when using DD4hep (in particular for the ODD) defining a ``matDeco`` in the main function is not strictly necessary as the DD4hep geometry can hold the information of which surface to map onto with which binning. We will ignore this option, since the goal of this guide is to explain how to make a material map regardless of the detector.
 
 As an output you will obtain the material map as a root and JSON file and a new material track collection in a root file. This new collection adds to each material interaction the associated surface during the mapping. This can be used for the control plots.
 Depending on what you want to do there are three options you can change:
 
-- ``mat-mapping-surfaces``: determine if material is mapped onto surfaces
-- ``mat-mapping-volumes``: determine if material is mapped onto volumes
-- ``mat-mapping-volume-stepsize``: determine the step size used in the sampling of the volume. This should be small compared to the bin size.
-
+- ``mapSurface``: determine if material is mapped onto surfaces
+- ``mapVolume``: determine if material is mapped onto volumes
+- ``mappingStep``: determine the step size used in the sampling of the volume. This should be small compared to the bin size.
+- ``readCachedSurfaceInformation`` if added the material-surface association will be taken from the input material track file (doesn't work with geantino file, you need to use the material track file obtained from running the material mapping).
```

Comment:
Considering you are mentioning the options used in the material mapping, can you also document here what the `mappingStep` refers to?

Reply:
Sure I can go into a bit more detail

Reply:
Does this make sense to you : 

- ``mappingStep``: determine the step size used in the sampling of the volume. The material interaction point obtained though G4 are only at the first intersection between the track and the material, this means that the mapping will be incorrect if the material is deeper than the bin size. To avoid this, we create new additional material point spaced equally every ``mappingStep`` along the trajectory. This should be small compared to the bin size.

Reply:
Does this parameter then make sense only for mapping onto volumes?
This is my suggestion, feel free to edit it further:
By default, the material interaction point obtained from G4 is accumulated at the intersection between the track and the volume material. The mapping will be therefore incorrect if the material extends through the bin. To avoid this, additional material points are created every mappingStep [mm] along the trajectory. The mapping step should be small compared to the bin size.

Reply:
Exactly it is only used in the volume mapping. The surface mapper doesn't use this parameters. Thanks for the  suggestion !

Reply:
I have updated the doc. It seems that now we have an issue in the CI related to the codding guideline page.